libs/frontend/util-render/src/lib/util-render.ts

```diff
@@ -52,15 +53,25 @@ export const queryStyleToString = (
   queryStyle: IQueryStyle,
 ): string => {
   let styleContent = ''
-  Object.entries(context.breakpoints).forEach(([breakpointId, breakpoint]) => {
-    const rawStyle = rawStyleRecordToString(queryStyle[breakpointId], context)
-    if (breakpointId === DEFAULT_BREAKPOINT_ID) {
-      styleContent += rawStyle
-    } else {
-      const mediaQuery = getMediaQuery(breakpoint)
-      styleContent += `@media ${mediaQuery} {${rawStyle}}`
-    }
-  })
+  Object.entries(context.breakpoints)
```

Comment:
@abemscac I had to make this change for the styles to be correctly ordered in preview/live mode. Otherwise, some of my sites would show correctly in the editor, but the smaller breakpoint styles would be overridden by the default/larger styles.

This can be optimized a lot, for example the `reverse` isn't necessary, but I was busy with other things. I also wonder if it's just hiding another bug, since I thought this worked before. It might have only happened after editing breakpoints, I'm not sure.